helm/helm_indent_finder.go

```diff
@@ -0,0 +1,198 @@
+package helm
```

Comment:
most packages in this repo are named [package]utils. Can we update just to follow that paradigm?